scripts/dist-git/remove_unused_sources

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/bash
+
+if [[ $# != 3 ]]; then
+    echo "invalid number of arguments"
+    exit 1
+fi
+
+package="$1"
+lookaside_dir="$2"
+git_dir="$3"
+
+pkg_git_dir="$git_dir/$package.git"
+pkg_dir="$lookaside_dir/$package"
+
+if [ ! -d "$pkg_dir" ]; then
+    echo "$pkg_dir is not a valid directory."
+    exit 1
+fi
+if [ ! -d "$pkg_git_dir" ]; then
+    echo "$pkg_git_dir is not a valid directory."
+    exit 1
+fi
+
+pushd /tmp > /dev/null
+
+git clone "$pkg_git_dir" 2> /dev/null
+cloned_dir=`basename "$package"`
```

Comment:
Ah, this is IMO not needed -- you can use the `git show REF:sources` in original repository.